homework.js

```diff
@@ -0,0 +1,118 @@
+//1. I tried but does not work !!  :(
+
+    const summatiom = function(n) {
+        if(n == 0) {
+            return 0;
+        } else {
+            return n - 1 + n
```

Comment:
change `return n - 1 + n` with  `return summatiom( n - 1) + n`, this will be a **recursive** way to solve the problem.
also you can always use a loop to solve this problem.